vmdb/config/vmdb.tmpl.yml

```diff
@@ -70,11 +70,15 @@ ems_refresh:
     get_shared_images: true
     get_public_images: false
     ignore_terminated_instances: true
-  scvmm:
+  foreman_configuration:
     :refresh_interval: 15.minutes
+  foreman_provisioning:
+    :refresh_interval: 1.hour
```

Comment:
These values shouldn't change often. And if `foreman_configuration` needs something else from `foreman_provisioning` it will kick off a refresh. So I was thinking mayeb `2.hours` @brandondunne / @gmcculloug did you guys already discuss this?